thrift/events.thrift

```diff
@@ -8,15 +10,15 @@ struct ContentAtomEvent {
    * across all content-atoms of any type (an alternative model might
    * be to have the `type+id` be unique) */
 
-   1: ContentAtomID id
+   1: required ContentAtomID id
 
    // this is a canonical place from which the current version of the
    //content atom's data can be downloaded.
-   2: string url
+   2: required string url
```

Comment:
It might not be needed you are right. But this is connected only to the event message, so although it might change, it is only intended to be correct for the lifetime of the event message, which in reality would be measured in minutes if not less.

Also, if we are going to have a contract for content-atom types that says they should be able to receive lifecycle events then they might need a URL endpoint to which these events can be posted, but that of course entirely depends on how we would implement that contract. It might just be that they should be listening to a kinesis stream etc.